.github/workflows/tpu-tests.yml

```diff
@@ -63,7 +63,7 @@ jobs:
           pip install mkl==2021.4.0 requests gsutil
 
           ## Install torch & xla
-          curl https://raw.githubusercontent.com/pytorch/xla/master/contrib/scripts/env-setup.py -o pytorch-xla-env-setup.py
+          curl https://raw.githubusercontent.com/abhi-glitchhg/ignite_tpu_script/main/env_setup.py -o pytorch-xla-env-setup.py
```

Comment:
@abhi-glitchhg we can't use your copy of `env_setup.py` as a solution. What's the difference between two files?

Reply:
OK, I get the difference, you removed `m` from the wheel name: `cp38-cp38m` -> `cp38-cp38`.
One hacky solution we can do is download original file and update (remove `m`) in locally with `sed`. Another solution could be to change the installation process to the one recommended in colab. I think the second approach would be a bit better

Reply:
Hello @vfdev-5 , removing `m` from the links is working, but I wanted to know the significance of m in the earlier link and why it has been dropped for 3.8 wheels! To make sure that we are not doing anything wrong here. 

Regarding the env_setup.py script, yes, we shouldn't rely on personal repos :). 
I was checking if this way is working or not.

The first approach is fine with me, or we could have the env_setup.py script inside the ignite repo; this way, we would have more control over the installation in the future. 

> Another solution could be to change the installation process to the one recommended in colab.

If I understand correctly, we would be directly installing wheels with the `pip install` command in the second approach, right? (correct me if my understanding is wrong) . (IMO, the current env_setup.py script is overkill for nightly installs.)  so this approach is fine with me. 

Let me know. 
Thanks

Reply:
Concerning `m` flag I could find the following info: 

> Per [PEP 3149](https://www.python.org/dev/peps/pep-3149/), m indicates that the ABI in question is using [the pymalloc allocator](https://docs.python.org/3/c-api/memory.html#pymalloc):

- https://stackoverflow.com/questions/54097033/what-does-the-m-in-a-python-abi-tag-mean
- https://peps.python.org/pep-3149/

and that in py38 `m` flag is no longer appended (ref: https://github.com/pypa/pip/pull/6874)

> If I understand correctly, we would be directly installing wheels with the pip install command in the second approach, right? (correct me if my understanding is wrong) . (IMO, the current env_setup.py script is overkill for nightly installs.) so this approach is fine with me.

yeah, let's take this approach and simplify the installation process. So, let's check it in the CI and hope that it works.

Reply:
Thanks for the info; I should have searched about this on google. sorry for the trouble. 

>let's take this approach and simplify the installation process. So, let's check it in the CI and hope that it works.

 I will try this.